src/langchain_google_alloydb_pg/indexes.py

```diff
@@ -63,9 +63,9 @@ class ExactNearestNeighbor(BaseIndex):
 @dataclass
 class QueryOptions(ABC):
     @abstractmethod
-    def to_string(self) -> str:
-        """Convert index attributes to string."""
-        raise NotImplementedError("to_string method must be implemented by subclass")
+    def to_parameter(self) -> list[str]:
```

Comment:
We will need to keep the to_string method in order for this not to be a breaking change.